src/main.cpp

```diff
@@ -676,12 +690,19 @@ void tick_all_systems(const SysTick_s &current_system_tick)
 
 }
 
-void handle_ethernet_interface_comms()
+void handle_ethernet_interface_comms(const SysTick_s& systick, const hytech_msgs_MCUOutputData& out_msg)
```

Comment:
maybe we should move this elsewhere, maybe make a proper interface for this instead of just sticking this function in main

Reply:
not fixing for now, it doesnt add too much to main and most of the functionality is already in the protobuf message interface

Reply:
agree, this should be moved to its own interface, and these comments need to be in a format that is actually parseable by vscode